diff --git a/node_modules/node-hid/nodehid.js b/node_modules/node-hid/nodehid.js
index 4b645da..cc08f6e 100644
--- a/node_modules/node-hid/nodehid.js
+++ b/node_modules/node-hid/nodehid.js
@@ -16,13 +16,22 @@ function loadBinding() {
         if( os.platform() === 'linux' ) {
             // Linux defaults to hidraw
             if( !driverType || driverType === 'hidraw' ) {
-                binding = require('bindings')('HID_hidraw.node');
+                binding = require('bindings')({
+                    bindings: 'HID_hidraw.node',
+                    module_root: __dirname,
+                });
             } else {
-                binding = require('bindings')('HID.node');
+                binding = require('bindings')({
+                    bindings: 'HID.node',
+                    module_root: __dirname,
+                });
             }
         }
         else {
-            binding = require('bindings')('HID.node');
+            binding = require('bindings')({
+                bindings: 'HID.node',
+                    module_root: __dirname,
+                });
         }
     }
 }
