diff --git a/build.rs b/build.rs
index dae9682..1545e18 100644
--- a/build.rs
+++ b/build.rs
@@ -150,6 +150,7 @@ fn build_rocksdb() {
         config.define("OS_MACOSX", None);
         config.define("ROCKSDB_PLATFORM_POSIX", None);
         config.define("ROCKSDB_LIB_IO_POSIX", None);
+        config.flag_if_supported("-faligned-allocation");
     } else if target.contains("android") {
         config.define("OS_ANDROID", None);
         config.define("ROCKSDB_PLATFORM_POSIX", None);
